main.py

```diff
@@ -342,35 +185,16 @@ def main():
         type=int,
         help="metrics server port",
     )
-    parser.add_argument(
-        "--pod-expire-duration",
-        type=float,
-        default=7200.0,
-        help="expiration time for pods in seconds",
-    )
-    parser.add_argument(
-        "--pod-without-metadata-expire-duration",
-        type=float,
-        default=300.0,
-        help="expiration time for pods without metadata in seconds",
-    )
     args = parser.parse_args()
 
-    # config should never be this way but this is an explicit sanity check
-    assert args.pod_without_metadata_expire_duration <= args.pod_expire_duration
-
     logging.basicConfig(
         level=logging.DEBUG if args.debug else logging.INFO,
         format="%(asctime)s %(message)s",
         datefmt="%Y/%m/%d %H:%M:%S",
     )
     # turn down the super verbose library level logging
-    logging.getLogger("kubernetes").setLevel(logging.CRITICAL)
     logging.getLogger("pika").setLevel(logging.CRITICAL)
 
-    # NOTE the service account needs read access to pod info
-    load_kube_config(args.kubeconfig)
-
     params = pika.ConnectionParameters(
         host=args.rabbitmq_host,
         port=args.rabbitmq_port,
```

Comment:
Do we want any of the following to be variables we can change based on a kubeconfig?  i assume not.

```
        client_properties={"name": "data-sharing-service"},
        connection_attempts=3,
        retry_delay=10,
        heartbeat=900,
        blocked_connection_timeout=600,
```

Reply:
Seems unlikely for now - I don't have a good enough feel for most of those to want to configure them a ton. Then only thing I may want to do is lower the heartbeat... but I'd have to experiment some.